.github/workflows/MacOSbacksupport.yml

```diff
@@ -0,0 +1,40 @@
+name: Wheel build test
+on: push
```

Comment:
I don't think you want this to run on every push... But rather just one tagged commits?

Reply:
No I just did this for testing the script. I can only test it by running the action. Now it's only on publish.